src/screens/application/Preview.tsx

```diff
@@ -1,123 +1,76 @@
 import React from "react";
-import {
-  Box,
-  Heading,
-  useDisclosure,
-  Modal,
-  ModalOverlay,
-  ModalContent,
-  ModalHeader,
-  Divider,
-  ModalCloseButton,
-  ModalBody,
-  ModalFooter,
-  Stack,
-} from "@chakra-ui/react";
+import { Box, useDisclosure, Stack } from "@chakra-ui/react";
 import "../../assets/styles/App.css";
-import Footer from "../../components/common/Footer";
 import CustomDisableInput from "../../components/common/inputs/DisabledInput";
 import CommonButton from "../../components/common/button/Button";
 import Layout from "../../components/common/layout/Layout";
-import HeadingText from "../../components/common/layout/HeadingText";
 import { useNavigate } from "react-router-dom";
+import ConfirmationDialog from "../../components/common/ConfirmationDialog";
+import BottomBar from "../../components/common/layout/Bottombar";
```

Comment:
if you use <BottomBar /> in the layout component so why use here ist how 2 time in this screen

Reply:
Removed Bottombar component